packages/ui/src/molecules/markdown/markdown.tsx

```
@@ -1,34 +1,34 @@
 };
 
 export const MarkdownSuspense = ({
-  content,
+  children = '',
   components = ComponentMap,
   FallBack = <div>Loading...</div>,
 }: MarkdownProps & { FallBack?: React.ReactNode }) => {
   return (
     <Suspense fallback={FallBack}>
-      <Markdown content={content} components={components} />
+      <Markdown children={children} components={components} />
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)